docs/common-usage-examples.mdx

```diff
@@ -81,9 +83,9 @@ One common example for this is project management software such as Github, Jira,
 
 #### Usage example
 
-One easy solution is to have one different application per each project. While it's possible to do it this way, it comes with a few annoying limitations. First of all, the app portal only shows one application at a time, so managing the endpoints is annoying. But also, oftentimes you'd like to listen for events affecting multiple projects from the same endpoint, and splitting it to different applications makes this very cumbersome.
+One easy solution is to have one different application per project. While it's possible to do it this way, it comes with a few annoying limitations. First of all, the app portal only shows one application at a time, so managing the endpoints is difficult. But also, oftentimes you'd like to listen for events affecting multiple projects from the same endpoint, and splitting it *WHAT IS IT?* into different applications makes this very cumbersome.
```

Comment:
@tasn Can you clarify the *WHAT IS IT?* here? :smirk:

Reply:
Splitting it to use multiple applications per project.